### PR TITLE
Add possibilities to use custom system grafana user and group.

### PR DESCRIPTION
**Pull request by neb0t**
Added possibilities to use custom system grafana user and group for archive installation method.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -364,38 +364,11 @@ sysconfig => {
 }
 ```
 
-### Advanced usage
```
Comment (bastelfreak) [inline README.md]: why was that removed?

File: manifests/config.pp [hunk 2]
```diff
@@ -12,8 +12,8 @@
         file {  $::grafana::cfg_location:
           ensure  => file,
           content => template('grafana/config.ini.erb'),
-          owner   => 'grafana',
-          group   => 'grafana',
+          owner   => $::grafana::grafana_system_user,
+          group   => $::grafana::grafana_system_group,
```
Comment (bastelfreak) [inline manifests/config.pp]: Please don't add `::` after `$`. It was required for puppet 3. We slowly try to get rid of it.

File: manifests/init.pp [hunk 3]
```diff
@@ -131,6 +137,8 @@
   Boolean $create_subdirs_provisioning  = $::grafana::params::create_subdirs_provisioning,
   Optional[String] $sysconfig_location  = $::grafana::params::sysconfig_location,
   Optional[Hash] $sysconfig             = undef,
+  String $grafana_system_user           = $::grafana::params::grafana_system_user,
+  String $grafana_system_group          = $::grafana::params::grafana_system_group,
```
Comment (bastelfreak) [inline manifests/init.pp]: please use `String[1]`, that will disable empty strings.

File: manifests/params.pp [hunk 4]
```diff
@@ -4,48 +4,50 @@
...
-  $docker_image                  = 'grafana/grafana'
-  $docker_ports                  = '3000:3000'
-  $install_dir                   = '/usr/share/grafana'
-  $package_name                  = 'grafana'
-  $rpm_iteration                 = '1'
-  $repo_name                     = 'stable'
-  $version                       = '4.5.1'
-  $create_subdirs_provisioning   = false
-  $provisioning_dashboards_file  = '/etc/grafana/provisioning/dashboards/puppetprovisioned.yaml'
+  $cfg = {}
```
Comment (bastelfreak) [inline manifests/params.pp]: please do nor change the alignment of the `=`

**Comment by dhoppe** [comment]
@neb0t Could please do a rebase? This pull request is tested against old versions of Puppet. For example, we dropped support for Puppet 4 two months ago.
